### PR TITLE
fix CI: macos was using proj8, which isn't supported

### PR DESCRIPTION
**Pull request by michaelkirk**
proj8 seems to mostly work, but gives slightly different results. I'm not sure if this is a proj regression, or maybe "better" calculations from proj, or perhaps some problem with our integration.

edit: also took the opportunity to (update CI to the latest rust).

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -29,7 +29,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        container_image: ["georust/proj-ci:rust-1.49", "georust/proj-ci:rust-1.50"]
+        container_image: ["georust/proj-ci:rust-1.49", "georust/proj-ci:rust-1.53"]
```
Comment (michaelkirk) [inline .github/workflows/test.yml:32]: I figured I'd leave 1.49 in the build matrix as some sort of notional minimum supported rust version.

**Comment by lnicola** [comment]
Can we add `georust/proj-ci:stable` tags to avoid bumping these every time?

**Comment by michaelkirk** [comment]
> Can we add georust/proj-ci:stable tags to avoid bumping these every time?

I think you've mentioned this before. You're welcome to propose this change to our ci docs and code which live here https://github.com/georust/docker-images. In the meanwhile I'm going to keep doing it this dumb way because bumping the tags is literally the easiest part.

The harder part is tracing down failures... like the fact that the macos package manager has updated their libproj beyond what we are testing against.

And now rust-1.53 is failing on ubuntu. Is this due to changes in rust 1.53 or the underlying ubuntu image? I don't know yet, but I don't think having an opaque moving tag like rust:latest would make any of that easier or solve a problem that I'm actually having.

**Comment by lnicola** [comment]
> I think you've mentioned this before.

Oh, sorry. I'll try to not say it a third time, but please ignore me if I do :confused:.

**Comment by michaelkirk** [comment]
superceded by #89

**Pull request closed without merging**